modules/text2vec-aws/clients/aws.go

```diff
@@ -144,39 +150,99 @@ func (v *aws) vectorize(ctx context.Context, input []string, operation operation
 	if err != nil {
 		return nil, errors.Wrapf(err, "AWS Secret Key")
 	}
+	awsSessionToken, err := v.getAwsSessionToken(ctx)
+	if err != nil {
+		return nil, err
+	}
 
-	headers["host"] = host
-	amzDate, headers, authorizationHeader := getAuthHeader(accessKey, secretKey, host, service, region, path, body, headers)
-	headers["Authorization"] = authorizationHeader
-	headers["x-amz-date"] = amzDate
+	maxRetries := 5
```

Comment:
is the AWS service unstable?
The problem with retrying is that we qill quickly run into timeouts between client and weaviate

Reply:
This is part of the legacy implementation, I just took this number out to some variable so that the same value will be used with `Bedrock` and `Sagemaker`